### ISSUE TITLE
Product families may inherit settings (attributes) from other family

### ISSUE THREAD

**Issue by wucherpfennig** [akeneo/pim-community-dev#6117]
**I'm submitting a Feature Request**

imho this would make sense, since you usually have "standard" attributes for each product whether it is a "book" or a "movie" or a "camera"...

**Comment by mickaelandrieu** [comment]
Hello @wucherpfennig,

thanks for your feedback: would you mind to elaborate your feature request a little bit please?

Regards,

**Comment by mickaelandrieu** [comment]
Hello @wucherpfennig,

if you want to contribute, feel free to open a new issue with more elaborated feature request.

Thank you for understanding

**Comment by LaureBro** [comment]
Hello @wucherpfennig, 
we have discussed with the Product team about your request and such a feature is already in our backlog under the reference IM-680. We've added a link to this Github issue. We'll keep you updated.
If you have any other need to raise us, feel free to raise a new issue and like @mickaelandrieu said, it's very important to have a use case with examples to better understand what you would like :)
Thanks!
